zombies.js

```diff
@@ -8,6 +9,21 @@
  * @property {string} name
  */
 
+class Item {
+  constructor(name) {
+    typeErrors(name, 'string');
+    this.name = name;
+  }
+}
+
+function typeErrors(objectToCheck, typeString) {
```

Comment:
For variable naming I think something like 'inputToCheck' might work better. By saying object the initial expectation is that you would be taking in an object.